certified-connectors/Replicon/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2403 @@
+                          "estimatedHoursToApply": {
+                            "type": "object",
+                            "description": "estimated hours to apply"
+                          },
+                          "estimatedCostToApply": {
+                            "type": "object",
+                            "description": "estimated cost to apply"
+                          },
+                          "resourceAssignmentModifications": {
+                            "type": "object",
+                            "properties": {
+                              "resourcesToAdd": {
+                                "type": "array",
+                                "items": {
+                                  "type": "string"
+                                },
+                                "description": "resources to add"
+                              },
+                              "resourcesToRemove": {
+                                "type": "array",
+                                "items": {
+                                  "type": "string"
+                                },
+                                "description": "resources to remove"
+                              }
+                            },
+                            "description": "resource assignment modifications"
+                          },
+                          "customFieldsToApply": {
+                            "type": "array",
+                            "items": {},
+                            "description": "custom fields to apply"
+                          },
+                          "keyValuesToApply": {
+                            "type": "array",
+                            "items": {},
+                            "description": "key values to apply"
+                          },
+                          "objectExtensionFieldsToApply": {
+                            "type": "array",
+                            "items": {},
+                            "description": "object extension fields to apply"
+                          }
+                        },
+                        "description": "task modification to apply"
+                      }
+                    }
+                  },
+                  "description": "task hierarchy"
+                },
+                "taskModificationOptionUri": {
+                  "type": "string",
+                  "description": "task modification option uri"
+                },
+                "unitOfWorkId": {
+                  "type": "string",
+                  "description": "unit of work id"
+                }
+              }
+            }
+          }
+        ],
+        "description": "Create Task Hierarchy Or Apply Modifications to a Replicon Task"
+      }
+    },
+    "/services/TaskService1.svc/MoveTask": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
```

Comment:
@repl-anand-vikram Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.